tests/yast2_gui/yast2_control_center.pm

```diff
@@ -136,13 +154,13 @@ sub start_authentication_server {
     assert_and_click 'yast2_control-center_authentication-server';
     assert_screen [qw(yast2_control-center-authentication-server_install yast2_control-center-authentication-server_configuration)], 90;
     if (match_has_tag('yast2_control-center-authentication-server_install')) {
-        send_key 'alt-i';
+        wait_still_screen { send_key 'alt-i' };
```

Comment:
why calling `wait_still_screen` when there is an `assert_screen` just afterwards?

Reply:
Makes sense